Application/Commands/Invitations/GetInvitationDetails/GetInvitationDetailsRequestHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Application.Common.Exceptions;
+using Application.Interfaces;
+using Application.Models;
+using Application.Providers;
+using AutoMapper;
+using MediatR;
+
+namespace Application.Commands.Invitations.GetInvitationDetails
+{
+    public class GetInvitationRequestHandler : RequestHandlerBase, IRequestHandler<GetInvitationDetailsRequest, InvitationDetailsDto>
+    {
+        public GetInvitationRequestHandler(IAppDbContext context, IAuthorizedUserProvider userProvider, IMapper mapper) : base(context, userProvider, mapper)
+        {
+        }
+
+        public async Task<InvitationDetailsDto> Handle(GetInvitationDetailsRequest request, CancellationToken cancellationToken)
+        {
+            Context.SetToken(cancellationToken);
+            Invitation invitation = await Context.Invitations.FindAsync(request.InvitationId);
+            if (invitation.ExpireTime < DateTime.Now)
```

Comment:
При создании приглашения можно не отправлять ExpireTime, так что тут его надо проверить на null
```suggestion
            if (invitation.ExpireTime is not null &&
                   invitation.ExpireTime < DateTime.Now)
```